vscode-dotty/package.json

```diff
@@ -49,8 +49,9 @@
     "tsc": "./node_modules/.bin/tsc",
     "vscode:prepublish": "npm run update-all && ./node_modules/.bin/tsc -p ./",
     "compile": "./node_modules/.bin/tsc -p ./",
-    "update-all": "npm install && node ./node_modules/vscode/bin/install && curl -L -o out/coursier https://github.com/coursier/coursier/raw/v1.0.0/coursier",
-    "test": "node ./node_modules/vscode/bin/test"
+    "update-all": "npm install",
```

Comment:
I think I would keep this as it was before, it matches how other vscode extensions do it.

Reply:
Do you know why it is done this way? This is exaclty the use case postinstall was made for.

Reply:
I just checked and the default template does use postinstall instead of update-all now actually, so this change is fine but we should get rid of update-all while we're at it.